portfolio/src/main/webapp/script.js

```diff
@@ -41,110 +41,159 @@ navbarList.setAttribute('class', 'navbar-list');
+  window.location.hash = 'gallery';
+
+  const wrapper = document.getElementById('wrapper');
+  wrapper.textContent = '';
+
+  const galleryTitle = document.createElement('h1');
+  galleryTitle.textContent = 'Gallery';
+  wrapper.appendChild(galleryTitle);
+
+  const galleryText = document.createElement('p');
+  galleryText.textContent = 'Photography sparks my interest - to me, taking a photo ' +
+    'means freezing a moment that would, otherwise, be impossible to reproduce. ' +
+    'Here are some photos I took or were taken of me:';
+  wrapper.appendChild(galleryText);
+
+  const imageContainer = document.createElement('div');
+  imageContainer.setAttribute('class', 'image-container')
+
+  const totalImagesNumber = 6;
+
+  for (let imageIndex = 1; imageIndex <= totalImagesNumber; imageIndex++) {
+    const imgUrl = `images/image-${imageIndex}.jpg`;
+    const imgElement = document.createElement('img');
+    imgElement.src = imgUrl;
+    imgElement.alt = 'Photo with/ by Raluca';
+    imageContainer.appendChild(imgElement);
+  }
+  wrapper.appendChild(imageContainer);
 }
 
-/**
- * Fetches comments from the server and adds them to the DOM.
- */
 function getComments() {
-    window.location.hash = 'comments';
-
-    const wrapper = document.getElementById('wrapper');
-    wrapper.textContent = '';
-
-    const commentsTitle = document.createElement('h1');
-    commentsTitle.textContent = 'Comments';
-    wrapper.appendChild(commentsTitle);
-
-    const newCommentButton = document.createElement('button');
-    newCommentButton.textContent = 'Add New Comment';
-    newCommentButton.onclick = getCommentForm;
-    wrapper.appendChild(newCommentButton);
-
-    const commnentContainer = document.createElement('ul');
-    fetch('/data').then(response => response.json()).then((comments) => {
-        for (let comment of comments) {
-            commnentContainer.appendChild(
-                createListElement(`${comment.text}, by ${comment.author}, \
-                                   posted on: ${comment.date}`));
-        }
-    });
-    
-    wrapper.appendChild(commnentContainer);
+  window.location.hash = 'comments';
+
+  const wrapper = document.getElementById('wrapper');
+  wrapper.textContent = '';
+
+  // Add "Comments" Heading
+  const commentsTitle = document.createElement('h1');
+  commentsTitle.textContent = 'Comments';
+  wrapper.appendChild(commentsTitle);
+
+  // Add button that redirects to the form for adding a new comment
+  const newCommentButton = document.createElement('button');
+  newCommentButton.textContent = 'Add New Comment';
+  newCommentButton.onclick = getCommentForm;
+  wrapper.appendChild(newCommentButton);
+
+  // Add form where the user can pick a maximum number of comments to fetch and display
+  const maxCommentsForm = document.createElement('div');
+  maxCommentsForm.setAttribute('id', 'max-comments-form-container');
+  // $(function() is a shorthand for jQuery(document).ready(function()
+  $(function() {
```

Comment:
You could use an arrow function instead: 
$(() => { ...

But not feeling strongly here.

Reply:
I tried this and it gave me an error in the console :(

Reply:
After adding a HTML file for the static elements of the 'Comments section', I got rid of these lines